mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -5962,3 +6055,25 @@ func containerClusterSurgeSettingsCustomizeDiff(_ context.Context, d *schema.Res
 
 	return nil
 }
+
+func containerClusterEnableK8sBetaApisCustomizeDiff(_ context.Context, d *schema.ResourceDiff, meta interface{}) error {
```

Comment:
Please add unit tests for this CustomizeDiff similar to https://github.com/hashicorp/terraform-provider-google/blob/f423e7c73a54d2f1a68bbfa394db7364a3f85f04/google/services/compute/resource_compute_instance_template_internal_test.go#L156

Reply:
Added unit tests, 715c029967fee43ddfde205490b1b89a587531f9 is required to pass all the test cases.